lib/pbench/agent/utils.py

```diff
@@ -210,6 +210,41 @@ class RedisServerCommon(BaseServer):
     local_host = None
 
 
+def check_failed_tool_init(tms: dict) -> str:
```

Comment:
Can we consider adding python unit tests for this new method?

Reply:
Based on https://github.com/riya-17/pbench/pull/5 we no longer need these methods add to the `utils` module in order for them to be shared between the Tool Data Sink code (which should not be applying policy) and `pbench-tool-meister-start`.

There is a refactoring PR for `pbench-tool-meister-start` in PR #2741 where the suggested behavior from https://github.com/riya-17/pbench/pull/5 would also be moved there, but I'd rather see this PR land with as few refactorings as possible.

Let's move this kind of improvement to another PR that layers on top of PR #2741.